array.go

```diff
@@ -66,6 +66,15 @@ func (a *Array) Raw() []interface{} {
 	return a.value
 }
 
+func (a *Array) Decode(target *[]interface{}) {
```

Comment:
* let's return receiver from this method; all httpexpect methods are chainable
* as I mentioned in chat, let's replace pointer to slice of interfaces with just interface; this will allow to pass pointer to slice of any objects, including structs, and current signature allows only slice of interfaces

Reply:
> * as I mentioned in chat, let's replace pointer to slice of interfaces with just interface; this will allow to pass pointer to slice of any objects, including structs, and current signature allows only slice of interfaces
Oh, yes I remember. I did  that you couldn't pass a reference for the target variable with just slice of interface, now changed it to just interface

Reply:
> let's return receiver from this method; all httpexpect methods are chainable
I don't quite understand this. receiver meaning target?

Reply:
Yes, exactly.